src/taskpane/taskpane.ts

```diff
@@ -4,15 +4,20 @@
  */
 
 import * as OfficeHelpers from "@microsoft/office-js-helpers";
+const testFunctions = ['=CONTOSO.ADD(5,2)', '=CONTOSO.CLOCK()', '=CONTOSO.INCREMENT(4)', '=CONTOSO.LOG("this is a test")'];
```

Comment:
Let's move the test code into a separate file `src\taskpane\test.ts` and have the Office.initialize code call test.runTests().